### PR TITLE
Refine guidance on event handling in “Introduction to events”

### PR DESCRIPTION
**Pull request by sideshowbarker**
Fixes https://github.com/mdn/content/issues/7955

### CODE REVIEW

**Comment by sideshowbarker** [comment]
I just now noticed that there’s also some odd problem with the sectioning hierarchy in this article – so I moved this PR to Draft status until I can identify the cause of that and fix it.

**Comment by sideshowbarker** [comment]
> I just now noticed that there’s also some odd problem with the sectioning hierarchy in this article – so I moved this PR to Draft status until I can identify the cause of that and fix it.

OK, fixed now — so this is back to being ready for review.

**Review by teoli2003 — APPROVED** [review]

File: files/en-us/learn/javascript/building_blocks/events/index.html [hunk 1]
```diff
@@ -240,26 +240,28 @@ <h3 id="adding_and_removing_event_handlers">Adding and removing event handlers</
 
 <h3 id="What_mechanism_should_I_use">What mechanism should I use?</h3>
 
-<p>Of the three mechanisms, you shouldn't use the HTML event handler attributes — these are outdated, and bad practice, as mentioned above.</p>
+<p>You shouldn't use the HTML event handler attributes — those are outdated, and bad practice, as mentioned above.</p>
 
-<p>The other two are relatively interchangeable, at least for simple uses:</p>
+<p>So for handling events, there are two methods you should consider:</p>
```
Comment (hamishwillee) [inline files/en-us/learn/javascript/building_blocks/events/index.html:245]: I was too late to review this but definitively an improvement sorry.

FWIW I would have done one thing slightly differently
1. Move the "You shouldn't use the HTML event ..." right to the bottom. We don't want them to be used, let's push them out of sight.
2. This section might then start with "There are only two methods you should consider for handling events:

Comment (sideshowbarker) [reply files/en-us/learn/javascript/building_blocks/events/index.html:245]: > FWIW I would have done one thing slightly differently
> 
>     1. Move the "You shouldn't use the HTML event ..." right to the bottom. We don't want them to be used, let's push them out of sight.
> 
>     2. This section might then start with "There are only two methods you should consider for handling events:

Agreed. Do you think it’s worth doing a follow-up PR to make that change? If so, do you want to do it? If not, I’d be happy to.

Comment (hamishwillee) [reply files/en-us/learn/javascript/building_blocks/events/index.html:245]: I think so. Done. https://github.com/mdn/content/pull/7990

**Pull request merged**